community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/birk/il/ExpandC.scala

```diff
@@ -22,35 +22,21 @@ package org.neo4j.cypher.internal.compiler.v2_3.birk.il
 import org.neo4j.graphdb.Direction
 
 case class ExpandC(fromVar: String, relVar: String, dir: Direction, types: Map[String, String], toVar: String, inner: Instruction) extends LoopDataGenerator {
-  private val visitMethod =
+  private val theBody =
```

Comment:
I really like this change, but I believe it might not really belong in this commit, right?